### ISSUE TITLE
Add UAC/User Account Control Support

### ISSUE THREAD

**Issue by Phorofor** [microsoft/winfile#79]
Add support for UAC when attempting to modify files and folders that require administrator privileges. 

Attempting to delete files or folders without the appropriate permissions for the current user brings up an 'Access is denied' dialog.

**Comment by ZanderBrown** [comment]
I think running the whole program as admin is the best option here

**Comment by NazmusLabs** [comment]
@ZanderBrown that should only be a stopgap solution. It's unwise to always have the entire a application elevated.

We can get my this much more easily.

For instance, the associate fiction. Is useless in Windows 10 even with admin priv. Windows 10 does not let applications change file association. Only users can do so. This is to prevent Spyware and aware changing which apps files open in. If an application changes association, the next time you open the file, a flyout comes up for the user to open it with the existing association or change the association.

It'd be better to have the associate feature just call the native association pop up, which would solve the problem in Windows 10 and solve the admin requirements for the associate feature in 7, 8, and 8.1.

**Comment by NazmusLabs** [comment]
Would anyone be willing to list all the functions in WinFile that gives access is denied error when not running elevated.

I have a couple:
* Associate
* Format disk (gives a different error)

Both of these functions are better off being redirected to their native OS counterparts when invoked.

**Comment by ZanderBrown** [comment]
@clzls on Windows 10?

**Comment by clzls** [comment]
@ZanderBrown definitely. But acquires admin priv. Almost everything here needs that. (Smile.)
